packages/styles/scss/themes/expressive/components/_ui-shell.scss

```diff
@@ -34,4 +34,19 @@
   .#{$prefix}--switcher__item-link {
     width: 100%;
   }
+
+  .#{$prefix}--header__action {
+    padding: 0;
+
+    &--active {
+      border: rem(2px) solid transparent;
+      outline: none;
+    }
+  }
+
+  .#{$prefix}--btn.#{$prefix}--btn--icon-only.#{$prefix}--tooltip__trigger:focus {
+    outline: none;
+    border-color: #f4f4f4;
```

Comment:
We don't have a token for this?

Reply:
Yep, I just found it, `$ui-01`. Just pushed a commit using it.

Reply:
Yeah, I removed the blue focus since that didn't seem to be present in the [current Expressive example](http://carbon-expressive.mybluemix.net/?path=/story/ui-shell--header-base-w-actions-and-right-panel). 

Let me know what's the proper way to handle the new focus

Reply:
Previously we've tried to maintain as much parody with the changes that come from Carbon directly so that we are not maintaining unnecessary overrides on our end. _Unless_ something is very broken or not accessible for whatever reason, and if/when that happens we can open bugs in their repo to fix something too. 

I don't have insight into exact reasons why they altered the focus state to include the blue+white, but we've previously trusted the updates to be correct.

I hope that makes sense!

Reply:
Alright, @shixiedesign just checked with them and apparently it's a 🐛 

So removing the blue focus was the correct move, ignore above comments.